src/components/SideNavBar/SideNavBar.jsx

```diff
@@ -1,22 +1,69 @@
 import React, { Component, PropTypes } from 'react';
+import { forEach } from 'lodash';
 
-import Base from '../Base.jsx';
+import getDataAttrs from '../../getDataAttrs';
+import SideNavBarItemGroup from './SideNavBarItemGroup';
+import SideNavBarTop from './SideNavBarTop';
 import style from './style.scss';
 
 class SideNavBar extends Component {
+
+  constructor() {
+    super();
+    this.renderItems = this.renderItems.bind(this);
```

Comment:
no need to do this if use enable `transform-class-properties` in babel, like [this](https://github.com/WeConnect/spacestation-v2/blob/master/.babelrc#L6)

```jsx
class SideNavBar extends Component {
  // this will automatically be bound to the component instance
  renderItems = () => {
    ...
  }
  ...
}